bottlerocket/samples/Makefile.toml

```diff
@@ -0,0 +1,242 @@
+[config]
+skip_core_tasks = true
+
+[env]
+AGENT_IMAGE_REGISTRY = { value = "public.ecr.aws/bottlerocket-test-system", condition = { env_not_set = ["AGENT_IMAGE_REGISTRY"] } }
+AGENTS = ["eks-resource-agent"]
+METADATA_BASE_URL = { value = "https://updates.bottlerocket.aws/2020-07-07", condition = { env_not_set = ["METADATA_BASE_URL"] } }
+ARCHITECTURE = { value = "x86_64", condition = { env_not_set = ["ARCHITECTURE"] } }
```

Comment:
nit: We use `ARCH` _everywhere_ else, so we probably should here.

Reply:
That would mean I'd have to change it in all sample files as well. Not a problem, just want to confirm that that's what you want?

Reply:
I guess we can change it in a follow up pr.

Reply:
I almost made the same comment. For better or worse we have standardized on `ARCH` in the Bottlerocket project and in other places within this project.

Reply:
It'll be changed in a follow-up PR. I'd rather not overload this one by changing 12 more files.